practice-javascript/src/scripts/services/userService.js

```diff
@@ -1,59 +1,59 @@
-import { API } from "../constants/url";
+import { API } from '../constants/url';
 
 export default class UserService {
   //function get user from json
-  static getUser = async () => {
+  static getAllUser = async () => {
     try {
       const res = await fetch(`${API.BASE_URL}${API.USER}`, {
```

Comment:
Please define the variable  for BASE_URL and USER to reuse instead of API.BASE_URL and API.USER